install/charts/templates/deployment.yaml

```diff
@@ -68,7 +61,18 @@ spec:
         imagePullPolicy: Always
         args:
           - "--replicate-cluster=v1/pods"
+      initContainers:
+      - name: pre-install-{{ .Values.metadata.name }}
+        image: lachlanevenson/k8s-kubectl
```

Comment:
Not sure if we can simply use this image. Is there an official image with kubectl installed on hub.docker.com?

Reply:
As far as I can tell, there is no "official" image. However, the used image has 10M+ downloads and is provided with a MIT license. Thus, this should not be a problem.

However, we could also create a kubectl image based on Alpine ourself (would be the same as the image from lachlanevenson).

Reply:
As discussed in our meeting, this image should be fine for now. We will catch up on this at a later point.